args/src/main/java/org/openjdk/skara/args/MultiCommandParser.java

```diff
@@ -62,6 +62,11 @@ public Executable parse(String[] args) {
                 var forwardedArgs = Arrays.copyOfRange(args, 1, args.length);
                 return () -> p.main(forwardedArgs);
             }
+            if (!"git-webrev".equals(programName) && !"git webrev".equals(programName)) {
```

Comment:
I was expecting a more elegant solution to this. I would add a property/method on the Command interface that  MutliCommandParser can use to check this. Something like Command::defaultCommandExpected.

Reply:
> I was expecting a more elegant solution to this. I would add a property/method on the Command interface that MutliCommandParser can use to check this. Something like Command::defaultCommandExpected.

Good idea. Should we add it in this patch? Or we should file another issue to implement it?

Reply:
This was meant as a suggestion for this patch. I don't think the current solution is ok to go in.

I'm not sure about the name of the new method, it's hard to come up with a good name for this property of a command. It's trying to express if the default sub-command is expected to be run with arguments or not, or if it takes arguments at all, but putting all that in a method name becomes quite a mouthful.

Reply:
What about `defaultCommandWarningEnabled`? Actually, the default sub-command is always invoked if no sub-command is provided. But the warning message are outputted optionally.

Reply:
What about `defaultCommandWarningEnabled`? Actually, the default sub-command is always invoked if no sub-command is provided. But the warning message are outputted optionally.

Reply:
Works for me. Thanks.

Reply:
I updated the code just now by using the property `defaultCommandWarningEnabled`.